Assignment4/src/main/java/com/example/Assignment4/entity/Items.java

```diff
@@ -0,0 +1,122 @@
+package com.example.Assignment4.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+/*
+ * pojo class of the item
+ */
+@Entity
+public class Items {
+
+
+    @Id
+    private String id;
+    private String name;
```

Comment:
-  Comments ?

Reply:
- @Table annotation is missing
- @ Column annotation is missing

Reply:
done